src/Apps/Search/Routes/Artworks/Components/Filter/Filters/PriceRangeFilter.tsx

```diff
@@ -1,25 +1,45 @@
 import { PriceRange } from "@artsy/palette"
 import { FilterState } from "Apps/Search/FilterState"
+import { track } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
 import React from "react"
 
-export const PriceRangeFilter: React.FC<{
+interface Props {
   filters: FilterState
-}> = ({ filters }) => {
-  const [initialMin, initialMax] = filters.rangeToTuple("price_range")
-  return (
-    <PriceRange
-      allowCross={false}
-      min={FilterState.MIN_PRICE}
-      max={FilterState.MAX_PRICE}
-      step={50}
-      defaultValue={[initialMin, initialMax]}
-      disabled={filters.state.at_auction}
-      onAfterChange={([min, max]) => {
-        const minStr = min === FilterState.MIN_PRICE ? "*" : min
-        const maxStr = max === FilterState.MAX_PRICE ? "*" : max
+}
+
+@track()
+export class PriceRangeFilter extends React.Component<Props> {
+  @track((_props: Props, _state, [price_range]) => {
+    return {
+      action_type: Schema.ActionType.ClickedCommercialFilter,
+      changed: { price_range },
+      current: { ..._props.filters.state },
```

Comment:
Same here

Reply:
oh yea it had the underscore from a previous version where i was passing filters in as a param, will update